content/iOS/RSFormView/RSFormView.md

```diff
@@ -0,0 +1,68 @@
+# Quickly Implementing elegant forms with RSFormView in Swift
+*[Germán Stábile](https://www.rootstrap.com/tech-blog/) is an iOS developer on the Rootstrap team and recently has started to write about some of the challenges faced during iOS development. You can also follow Germán on [LinkedIn](https://www.linkedin.com/in/german-stabile-61a1b755/).*
+
+Note: this post is meant to discuss motivations and implementation of RSFormView, if you are interested in a tutorial on how to integrate please visit [RSFormView Readme](https://github.com/rootstrap/RSFormView) or the next article [Link to next article].
+
+## Introduction 
+
+During my first experience with React-Native for a short, form-based project, I found extremely useful the [redux-form](https://github.com/erikras/redux-form)  library and kept thinking on how nice would it be to have such library for iOS development. After finishing that project it was time to move back to iOS native programming for another project (yay!).
+A few glances at the beutiful wireframes the design team provided and I see at least five forms for data entry: and so I thought, developing a form component with such beautiful design (by default) may be both useful and fun! 
+
+In my time as an iOS developer I have seen and work with a bunch of different form implementations, some repeating too much code and the ones with a more componetized solution were buggy or too difficult to extend or accomplish the design intended. 
+
+I wanted to avoid these issues and implement something we could use in different projects but giving the time constraints I knew I couldn't build something as configurable as [redux-form](https://github.com/erikras/redux-form).
+
+However with the beautiful design I was handed and making fonts, colors, and margins configurable it should be enough given that we can extend as we need more functionalities and maybe some day making it fully customizable. 
+
+Development on that project was very smooth having implemented this component before hand. I was able to finish any form based screen in a few minutes and with very few lines of code. Seeing how happy I was with the results I decided to use the extra time in my hands and share the goods with the open source community.    
+
+## What Is RSFormView?
+
+[RSFormView](https://github.com/rootstrap/RSFormView) is a library that allows you to create data entry forms in a few minutes with beautiful out of the box  design that you can also configure setting your custom colors, fonts and margins. 
+
+![Alt](images/formExample.gif)
+
+### What can it do? 
+
+- It supports different out of the box data entry options: regular text fields, password text fields, date pickers, options pickers, etc.
```

Comment:
I think `It supports  data entry options different out of the box` reads a bit nicer